container.go

```diff
@@ -43,6 +43,7 @@ type Container struct {
 	PidMode              string                        `yaml:"pid_mode" json:"pid_mode"`
 	ShmSize              int64                         `yaml:"shm_size" json:"shm_size"`
 	Labels               []string                      `yaml:"labels" json:"labels"`
+	StopTimeout          uint                          `yaml:"stop_timeout" json:"stop_timeout"`
```

Comment:
I think this should be signed so that -1 == never timeout, 0 == current default, everything else is the timeout.

Reply:
going to leave it as unsigned for now as we'll need to add a method for the docker client to stop containers without a timeout arg... thoughts @emosbaugh ?